src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -467,7 +467,11 @@ def makeJobSubmit(self, task):
         info['taskType'] = self.getDashboardTaskType(task)
         info['worker_name'] = getattr(self.config.TaskWorker, 'name', 'unknown')
         info['retry_aso'] = 1 if getattr(self.config.TaskWorker, 'retryOnASOFailures', True) else 0
-        info['aso_timeout'] = getattr(self.config.TaskWorker, 'ASOTimeout', 0)
+        if kwargs['task']['tm_output_lfn'].startswith('/store/user/rucio') or \
```

Comment:
I see that pylint complains about `kwargs` not being defined. maybe we should use directly `info`? something like

```diff
- kwargs['task']['tm_output_lfn'].startswith('/store/user/rucio') ...
+ info['tm_output_lfn'].startswith('/store/user/rucio') ...
```

?

Reply:
oopss... thanks @mapellidario  !!!! (and pylint)